certified-connectors/ForceManager CRM/apiDefinition.swagger.json

```diff
@@ -3318,8 +3312,9 @@
         },
         "birthday": {
           "type": "string",
+          "format": "date-time",
```

Comment:
Added format, can you please confirm that no existing flows or users will be affected by this change?

Please see the validation error below. 

##[error]The type format has been changed for the input field. Existing flows will continue to send the field with the old type format. ValuePath: operation/CreateContact/parameter/body/schema/birthday/format Source: NewSwagger

Reply:
Yes I confirm that no existing flows or users will be affected, since the connector still has to be officially released. We are in the process of validating it in a preview env of PA.